packages/ui/src/i18n/dictionaries/authenticator/de.ts

```diff
@@ -29,6 +29,7 @@ export const deDict: AuthenticatorDictionary = {
     'Es kann eine Minute dauern, bis er ankommt',
   Loading: 'Wird geladen',
   'New password': 'Neues Passwort',
+  or: 'oder',
```

Comment:
just curious, do we know if this will work with all language sentence structures? I could imagine that some languages may not use or in the same way in the middle of the sentence... Maybe they do, I only know English and Spanish...

Reply:
Good question. This specific string is used in isolation on the FederatedSignIn screen, so I don't think we need to be concerned about sentence structure. 

<img width="891" alt="AuthOr" src="https://user-images.githubusercontent.com/48109584/213569510-0387c883-f647-47f4-b02c-d93163efe9cf.png">